app/components/questions/ResidentsWithDisabilities.js

```diff
@@ -0,0 +1,145 @@
+import { useEffect } from 'react'
+import useFetch from 'use-http'
+import {
+  ResponsiveContainer,
+  PieChart,
+  Pie,
+  Cell,
+  Tooltip,
+  Legend,
+} from 'recharts'
+import { FailureNotification } from '../../components/Notification'
+import { Color } from '../../site/theme'
+import { Formatter } from '../../site/metrics'
+
+const ENDPOINT = `${process.env.NEXT_PUBLIC_API}/community/metrics`
+
+const TRANSIT_MODES = {
```

Comment:
We no longer need this variable, so you can delete lines 17-34.

Reply:
These lines have been removed